dev-test/metadata.test.js

```diff
@@ -66,7 +66,7 @@ describe("metadata examples", () => {
     const args = [[value], index];
 
     try {
-      const result = await executeScript({ code, args });
+      const [result] = await executeScript({ code, args });
       expect(result).toBe(value[index]);
     } catch (e) {
       console.error(e);
```

Comment:
We don't need to `try` here - we should be catching errors inside `executeScript` :)

Reply:
Fixed. To be thorough I went through everything again to assure that all use of both `executeScript` and `sendTransaction` allow the errors to be caught inside of each method respectively. They should all be correct now.